Runtime/Scripts/RGOverlayMenu.cs

```diff
@@ -173,9 +174,18 @@ public void AddBot()
 
         public void TempTestAddLocalBot()
         {
-            // start the local bot
-            // TODO (REG-1291): Test Code: This number is meaningless until we lookup bots locally by id
-            RGBotRuntimeManager.GetInstance()?.StartBot(-123456789L);
+            // TODO (REG-1291): Temporary code to get bot 0.. this whole method will go away in
```

Comment:
This will go away in what?

Reply:
That's what I thought but was double checking, my brain had a hard time parsing that comment 😂